charts/thoras/values.yaml

```diff
@@ -119,11 +144,23 @@ thorasDashboard:
 
 thorasMonitor:
   enabled: false
+  unittesting: false
```

Comment:
This is a workaround to the fact the monitor deployment loads the contents of configmap into the pod spec which makes unittesting impossible. With this flag we can turn that functionality off -- allowing us to "skip" the line that does that